Added blog for mentorship (#291)
pythonindia/inpycon-blog: content/blog/2020-pycon-india-mentorship.md

> Title: Want to perfect your proposal? The PyCon India mentorship team awaits you. 

Reviewer: ```suggestion
Title: Want to perfect your proposal? The PyCon India mentorship team is here to help you. 
```
how anout this?

Date: 2020-07-27 00:34 
Category: 2020 
Slug: pycon-india-mentorship 

> Summary: Want to perfect your proposal? The PyCon India mentorship team awaits you.

Reviewer: Is there a point in adding a summary same as the title?

Reviewer: Also, please add an Author for the article xD

Do you have that one idea that can make the world “a better place”? (Silicon Valley Fans, Assemble!) 
Do you wish to submit a proposal but cannot figure out the flow of your talk or how to present it? 
We know it is not always easy to find the right words, but don’t you worry, the PyCon India Mentorship team is here just to help out with your proposal. 

> Whether this is your first time or you are a seasoned professional who wants someone to take a look at their work, the PyCon India Mentorship team is here just to help you out with your proposal. 

Reviewer: The "PyCon India ....." section is the same as the previous line. 
Maybe try something like this
```suggestion
Whether this is your first time or you are a seasoned professional who wants someone to take a look at their work, our mentorship team will guide you through the process and help you wherever required. 
```

> Now there are quite a few ways to reach out to us and you are free to use any. 

Reviewer: ```suggestion
Now, there are quite a few ways to reach out to us and you are free to use any of them. 
```

Reviewer: Also, this line might look better after the blog reference.

The go-to resource for prospective speakers of PyCon India, you can have a look at one of the blogs shared on the website which shares [tips on writing a “kickass” proposal](https://in.pycon.org/blog/2020/2020-workshop-cfp-announcement.html). 

> In case you want to know the best practices of writing a proposal or are looking to refine your proposal, a [call](https://meet.google.com/pot-jzjq-udn) is scheduled every Saturday 1500 to 1700 IST which you can join and clear out your queries on a real-time basis. 

Reviewer: ```suggestion
In case you want to know the best practices of writing a proposal or are looking to refine your proposal, a [call](https://meet.google.com/pot-jzjq-udn) is scheduled every Saturday 1500 to 1700 IST by the mentorship team which you can join and clear out your queries on a real-time basis. 
```

If the next Saturday seems distant and you need immediate help, you can join the [PyCon India Zulip](https://pyconindia.zulipchat.com/#) and drop a text in the [pre-cfp topic](https://pyconindia.zulipchat.com/#narrow/stream/245487-wg.2Fmentorship/topic/pre-cfp) of the mentorship stream. 
You will get a response from the team and will be asked to share a Google document with comment access to a mentor(in a private message), who will then help you polish the draft. 
If you don't get any response in 48 hours, please send a direct message to @siddharthvipul. And that’s all you need to do. 

And Pythonista, we feel you when you find the entire process of framing a proposal difficult. 

> PyCon India is a community-driven event and our foremost priority is to learn and grow. 

Reviewer: ```suggestion
PyCon India is a community-driven event and our foremost priority is to help you learn and grow. 
```

> Title: Writing a proposal for the first time? Mentorship team is here to help.

Reviewer: our mentorship team?

<!-- PELICAN_END_SUMMARY -->

> Do you have that one idea that can make the world “a better place”? (Silicon Valley Fans, Assemble!)

Do you wish to submit a proposal to PyCon India 2020 but cannot figure out the flow of your talk or how to present it?

We know it's not always easy to find the right words when you have something to share. It happens to the best of us. But no need to worry, as the PyCon India mentorship team is here to help you out with your proposal. Whether you are submitting a proposal at PyCon India for the first time or you are a veteran speaker with years of experience behind you, our mentorship team will guide you through the process and help you wherever required in a number of ways as mentioned below. We want to help you put your best foot forward when it comes to submitting proposals! 
 
## Recommended Resources from the Content Team 

Prospective speakers of PyCon India can take a look at our curated blog posts on writing better talk and workshop proposals.  
- [Tips on writing a “kickass” proposal](https://in.pycon.org/blog/2020/2020-workshop-cfp-announcement.html) 

> - [Guidelines on creating video previews](https://in.pycon.org/blog/2020/2020-call-for-proposals-announcement.html)

Reviewer: This is not rendering properly https://deploy-preview-291--inpyconblog.netlify.app/2020/announce-pycon-india-mentorship.html

Author: I tried a different way of adding bullets in markdown but it still does not work.

Reviewer: You need at least 2 empty spaces after you text to line break propoerly

## Join us on Zulip for faster answers to any questions

> Moreover, our mentors will be in constant touch on Zulip. You can join them on the [PyCon India Zulip Channel](https://pyconindia.zulipchat.com/#) and drop a text in the [pre-cfp topic](https://pyconindia.zulipchat.com/#narrow/stream/245487-wg.2Fmentorship/topic/pre-cfp) of the #Mentorship stream (Streams are available to choose from the sidebar on the left). 

Reviewer: Should we start the section with Moreover?

How about `You can always contact our mentors on Zulip.`

## Open Mentorship calls 

In the hopes of providing better opportunities and assistance for first-time speakers at PyCon India 2020, we are starting weekly mentorship calls. This would be an open call for participants and potential speakers to interact with PyCon India mentors to review, share, and discuss draft proposals. Details of the call are as follows:

Date: Weekly on Saturdays 
Time: 15:00 to 17:00 IST

> Platform: Google Meet - Join with the [invite link](https://meet.google.com/pot-jzjq-udn)

Reviewer: Same here

Author: I added another line break between each of the lines. So it does seem better and is presented in new lines.
